dspace/config/modules/bitstore-srb.cfg

```diff
@@ -0,0 +1,57 @@
+##### SRB File Storage #####
```

Comment:
I wouldn't say the code has been removed, I would just say that SRB hasn't been implemented using this refactored bitstore. I emailed dspace-tech on June 5 2015 "DSpace Assetstore, anyone use SRB Storage Resource Broker?", and never got a response that SRB was in use with DSpace. So yes, following this logic, the stub for the config shouldn't be here if there is no SRB implementation. And we would note that this PR drops support for SRB assetstore.